incubator/kubeless/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-controller
+  namespace: kubeless
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+    kubeless: controller
```

Comment:
Is this label required for Kubeless? Otherwise it should be `component`.

Reply:
Yes, it is required